stackslib/src/net/relay.rs

```diff
@@ -2499,14 +2499,27 @@ impl Relayer {
                     for chunk in sync_result.chunks_to_store.into_iter() {
                         let md = chunk.get_slot_metadata();
                         if let Err(e) = tx.try_replace_chunk(&sc, &md, &chunk.data) {
-                            warn!(
-                                "Failed to store chunk for StackerDB";
-                                "stackerdb_contract_id" => &format!("{}", &sync_result.contract_id),
-                                "slot_id" => md.slot_id,
-                                "slot_version" => md.slot_version,
-                                "num_bytes" => chunk.data.len(),
-                                "error" => %e
-                            );
+                            if matches!(e, Error::StaleChunk { .. }) {
+                                // This is a common and expected error, so log it as a debug and with a sep message
+                                // to distinguish it from other error types.
```

Comment:
```suggestion
                                // This is a common and expected message, so log it as a debug and with a sep message
                                // to distinguish it from other message types.
```
i think it can be either way, but saying we're suppressing an error just sounds bad

Reply:
Agreed